lib/Utils/Backends.pm

```diff
@@ -262,7 +262,7 @@ sub is_generalhw { check_var('BACKEND', 'generalhw'); }
 #The ssh connection timeout is counted as seconds
 sub set_ssh_console_timeout {
     my ($sshd_config_file, $sshd_timeout) = @_;
-    $sshd_config_file //= '/etc/ssh/sshd_config';
+    $sshd_config_file = '/etc/ssh/sshd_config' if (!$sshd_config_file or script_run("ls $sshd_config_file") != 0);
```

Comment:
It would be worth to comment why we need to do such things, which in my opinion doesn't look nice.

Reply:
Comment can be added.